app/src/main/java/com/example/concordia_campus_guide/Activities/RoutesActivityViewModel.java

```diff
@@ -146,9 +149,9 @@ public void setTransportType(String transportType) {
         this.transportType = transportType;
     }
 
-    public void noShuttles() {
+    public void noShuttles(Context context) {
```

Comment:
Can this take the string to display as the parameter and avoid taking the context as a param?